crates/forge/bin/cmd/debug.rs

```diff
@@ -1,21 +1,21 @@
-use super::{build::BuildArgs, retry::RETRY_VERIFY_ON_CREATE, script::ScriptArgs};
+use super::{build::BuildArgs, retry::RetryArgs, script::ScriptArgs};
 use clap::{Parser, ValueHint};
-use foundry_cli::opts::CoreBuildArgs;
+use ethers::types::U256;
+use foundry_cli::{opts::MultiWallet, utils::parse_ether_value};
 use foundry_common::evm::EvmArgs;
-use std::path::PathBuf;
 
 // Loads project's figment and merges the build cli arguments into it
 foundry_config::impl_figment_convert!(DebugArgs, opts, evm_opts);
 
 /// CLI arguments for `forge debug`.
-#[derive(Debug, Clone, Parser)]
+#[derive(Debug, Clone, Parser, Default)]
 pub struct DebugArgs {
```

Comment:
hmm sorry for the bikeshed but slightly confused about this—wasn't the plan to `#[flatten]` `ScriptArgs`?

Reply:
Do you mean Serde's flatten ? Isn't this only for Serialising / Deserializing ? I meant `[#clap(flatten)]` for it to be seamless to the cli

Reply:
yup meant `#[clap(flatten)]`—i thought we were gonna do this instead of copying over all the script struct fields 😅

Reply:
I didn't copied all the struct field, look at `ScriptArgs` 😁

Reply:
@iFrostizz coming back to this, gotcha now i see. Frankly though this doesn't sit super well with me as we're now hiding a bunch of important args for script on `DebugArgs` and I'm not sure if this is good for flexibility. I think we can handle a bit of duplication for now—wdyt about duplicating the script args on debug? we'll be rewriting a few of these things over the coming months, so it's fine.

Reply:
Okay, this won't be very future proof tho